wikipedia-dark.user.css

```diff
@@ -2604,7 +2604,7 @@ regexp("https?:\/\/wiki\.(rpcs3\.net|(archlinux|mozilla)\.(org|jp))\/.*$") {
   .mw-rcfilters-ui-filterMenuOptionWidget.oo-ui-flaggedElement-muted:not(.oo-ui-optionWidget-selected) {
     background-color: var(--gray-2);
   }
-  div[style*="background-color:#f9f9f9"] {
+  div[style*="background-color:#f9f9f9"], div[style*="background-color:#fffaf5"] {
```

Comment:
It would be nice if that border was also fixed =)

Reply:
@the-j0k3r added something that I think could work, but can't test it